sdk/python/Makefile

```diff
@@ -59,4 +59,4 @@ test_fast::
 
 dist::
 	go install -ldflags "-X github.com/pulumi/pulumi/sdk/python/pkg/version.Version=${VERSION}" ${LANGHOST_PKG}
-	cp ./cmd/pulumi-language-python-exec "$(go env GOPATH)"/bin/
+	cp ./cmd/pulumi-language-python-exec $(GOPATH)/bin/
```

Comment:
Can you make this  `"$$(go env GOPATH)"/bin/` and see if that works?  I think that should do what you want (make will treat the $$ as a $ and then the shell will interpret it as expected.  Since GO 1.9, there's a default value for GOPATH if it is unset, and so it is common to not be set in the environment.

Reply:
Make sense, testing it right now.

Reply:
Working fine, updating 😄